<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCameraActuatorAssociation</class>
 <widget class="QDialog" name="DialogCameraActuatorAssociation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera to Actuator associations</string>
  </property>
  <widget class="QTreeView" name="treeView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>621</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Double click on a camera from the list to edit actuator(s) to associate:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_expandTreeView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Expand all</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_collapseTreeView">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Compress all</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>440</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
